src/templates/sermons.js

```diff
@@ -3,12 +3,30 @@ import Layout from "../components/layout"
 import './sermons.css'
 
 export default function Sermons({ pageContext }) {
-    const sermons = pageContext.nodes;
+    const sermons = pageContext.nodes,
+        {total, current} = pageContext.pagination,
+        pageStart = Math.max(1, current - 10),
+        pageEnd = Math.min(total, current + 10),
+        pagePrevious = Math.max(1, current - 1),
+        pageNext = Math.min(total, current + 1);
+
+    let pageLinks = []
```

Comment:
Since you're producing an array of page links from a range of numbers it might be simpler to use `_.range()` and `.map()`:

```js
const pageLinks = _.range(pageStart, pageEnd + 1).map((i) => 
    <li className={current === i ? "active" : null}>
        <a href={`/en/sermons/page/${i}`}>{i}</a>
    </li>);
```